### ISSUE TITLE
Unable to find version 5.0.0-rc01 as specified in docs

### ISSUE THREAD

**Issue by kvaithin** [neo4j-contrib/neo4j-apoc-procedures#3497]
## Guidelines

Please note that GitHub issues are only meant for bug reports/feature requests. If you have questions on how to use APOC, please ask on [the Neo4j Discussion Forum](https://community.neo4j.com/c/neo4j-graph-platform/procedures-apoc) instead of creating an issue here.

## Expected Behavior (Mandatory)

Looking to run Neo4j with Plugins via docker. 
The steps for using version 5.x involves running `wget` with following command. 

wget https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/5.0.0-rc01/apoc-5.0.0-rc01-extended.jar

Which should just download. 

See the docs here: https://neo4j.com/labs/apoc/5/installation/#docker

## Actual Behavior (Mandatory)

It doesn't download. Unable to find the path. 

When checked here. Seems like latest version is only 4.4. 

https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases

Path for version 5.0.0-rc01 is missing.

Believe either docs is incorrect or this version path needs to be updated. 

## How to Reproduce the Problem

Run the following command in terminal. 

wget https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/5.0.0-rc01/apoc-5.0.0-rc01-extended.jar

Will see following error. 

```
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-03-21 19:32:26 ERROR 404: Not Found.
```

### Steps (Mandatory)

  1. Open terminal (In a mac or linux)
  2. Run this command.
  wget https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/5.0.0-rc01/apoc-5.0.0-rc01-extended.jar

Currently used versions

### Versions

  - OS: Mac Ventura v13.0.1
  - Neo4j: 5.5.0
  - Neo4j-Apoc: 5.0.0-rc01 // not found

**Comment by gem-neo4j** [comment]
Hey! Sorry for the confusion, I'll make sure we get the docs updated :)

As you are using Neo4j version 5.5.0, you wouldn't be able to use APOC 5.0.0 anyway, you need to use APOC 5.5.0 to match (the major and minor must always match, and then we recommend if any patch exists to use the highest available for both APOC and Neo4j).

Also, just so you are aware, as of the 5.x series APOC is split in 2. The release you are trying to download is APOC extended, if you want APOC core as well, you will need to download APOC core, that can be found [here](https://github.com/neo4j/apoc/releases/tag/5.5.0), and the docs for that are [here](https://neo4j.com/docs/apoc/current/installation/). 

The correct link for APOC 5.5.0 extended is https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/5.5.0/apoc-5.5.0-extended.jar. The latest released jar is indeed 4.4, but that is a patch for the 4.4.x series, if you scroll down you will find the 5.x releases as well.

Hope this helps :)

**Comment by gem-neo4j** [comment]
The docs have been updated :)
